### PR TITLE
Prevent marines from pulling parasites, localize pull whitelists

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
fixes [discord issue](https://discord.com/channels/1168210010233376858/1269721418887725118)

https://github.com/user-attachments/assets/77d8b37c-ca10-424f-98a2-0b2a1bfc5969

### CODE REVIEW

File: Resources/Locale/en-US/_RMC14/pulling/pulling.ftl [hunk 1]
```diff
@@ -1,2 +1,8 @@
 cm-pull-whitelist-denied = We have no use for {THE($name)}, why would we want to touch it?
 cm-pull-whitelist-denied-dead = {THE($name)} is dead, why would we want to touch it?
+
+rmc-pull-paralyze-self = You try to pull {$pulled}, getting a tail swipe to the head!
+rmc-pull-paralyze-others = {$puller} tries to pull {$pulled}, but gets a tail swipe to the head instead!
+
+rmc-pull-infect-self = You try to pull {$pulled}, but get jumped on and infected in the process!
```
Comment (Dutch-VanDerLinde) [inline Resources/Locale/en-US/_RMC14/pulling/pulling.ftl:7]:
```suggestion
rmc-pull-infect-self = You try to pull {$pulled}, but gets jumped on and infected in the process!
```

File: Resources/Locale/en-US/_RMC14/pulling/pulling.ftl [hunk 2]
```diff
@@ -1,2 +1,8 @@
 cm-pull-whitelist-denied = We have no use for {THE($name)}, why would we want to touch it?
 cm-pull-whitelist-denied-dead = {THE($name)} is dead, why would we want to touch it?
+
+rmc-pull-paralyze-self = You try to pull {$pulled}, getting a tail swipe to the head!
+rmc-pull-paralyze-others = {$puller} tries to pull {$pulled}, but gets a tail swipe to the head instead!
+
+rmc-pull-infect-self = You try to pull {$pulled}, but gets jumped on and infected in the process!
```
Comment (Dutch-VanDerLinde) [inline Resources/Locale/en-US/_RMC14/pulling/pulling.ftl:7]:
```suggestion
rmc-pull-infect-self = You try to pull {$pulled}, but get jumped on and infected in the process!
```

File: Content.Shared/_RMC14/Pulling/CMPullingSystem.cs [hunk 3]
```diff
@@ -62,20 +65,39 @@ private void OnParalyzeOnPullAttempt(Entity<ParalyzeOnPullAttemptComponent> ent,
...
+            return;
         }
+
+        if (!TryComp<XenoParasiteComponent>(target, out var paraComp))
+            return;
+
+        Entity<XenoParasiteComponent> comp = (target, paraComp);
+        args.Cancelled = true;
+
+        _parasite.Infect(comp, user, false, true);
```
Comment (DrSmugleaf) [inline Content.Shared/_RMC14/Pulling/CMPullingSystem.cs:93]: This should check if Infect returned true before displaying the popup

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by Dutch-VanDerLinde** [comment]
Reviews done

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**